k8s/wordpress/apptest/tester/tests/exporter-suite.yaml

```diff
@@ -7,3 +8,35 @@ actions:
         matches: '(?m)^mysql_up 1$'
       exitCode:
         equals: 0
+
+# Some of metrics are exposed even if the exporter doesn't have access to the MySQL server statistics.
+# This tests verify that access is granted.
+- name: Can MySQL Exporter connect to the MySQL server
```

Comment:
Well... I've been thinking - maybe we should use HttpTest for scrapping those Prometheus endpoints? It seems reasonable to use that.

Reply:
Unfortunately we can not. Http test can't expect the content, only the title is supported.

Reply:
https://github.com/GoogleCloudPlatform/marketplace-testrunner/blob/master/specs/specs.go#L82-L84